Source/WebKit/Platform/spi/ios/PhotosUISPI.h

```diff
@@ -52,6 +56,20 @@
 
 @end
 
+@interface PHPickerConfiguration : NSObject
```

Comment:
This should be a category rather than an interface declaration, and should be moved outside of `USE(APPLE_INTERNAL_SDK)`.

The purpose of declaring SPI is for open source builds where `USE(APPLE_INTERNAL_SDK)` is false.

Reply:
@pxlcoder do you have an example of a category that accomplishes something similar to what this is supposed to do?

Reply:
There are many of them in `UIKitSPI.h` – `@interface UIDatePicker ()` is one example.